ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -87,6 +121,34 @@ int main()
     }
 }
 
+std::string getInput()
+{
+    std::string command;
+    std::getline(std::cin, command);
+    return command;
+}
+
+cv::Mat imageToRGB(const Zivid::Image<Zivid::RGBA8> &image)
+{
+    cv::Mat rgb(static_cast<int>(image.height()), static_cast<int>(image.width()), CV_8UC3, cv::Scalar(0, 0, 0));
+
+    const auto height = image.height();
+    const auto width = image.width();
+
+    for(int i = 0; i < height; i++)
+    {
+        for(int j = 0; j < width; j++)
+        {
+            cv::Vec3b &color = rgb.at<cv::Vec3b>(i, j);
```

Comment:
You can convert without iterating over the whole image. Try this

        cv::Mat rgbaMat(height(), width(), CV_8UC4, const_cast<void *>(reinterpret_cast<const void *>(image.dataPtr())));
        cv::Mat rgbMat;
        cvtColor(rgbaMat, rgbMat, cv::COLOR_RGBA2RGB);

Reply:
This should be much faster

Reply:
Except I would do:
cvtColor(rgbaMat, rgbMat, cv::COLOR_RGBA2BGR);

BGR instead of RGB (as OpenCV use BGR as default)

Reply:
...or modify the display function, and do the conversion there. All functions and variables indicate rgb, but `pointCloudToRGB` actually returns BGR. An alternative would be to rename all instances of rgb to bgr, where it's actually bgr.

Reply:
Using the same format throughout the entire sample seems like a good idea. As well as renaming the functions and variables to reflect the correct rgb/bgr.